src/shared/user-record-show.c

```diff
@@ -436,10 +436,13 @@ void user_record_show(UserRecord *hr, bool show_full_group_info) {
 
                 STRV_FOREACH(i, hr->pkcs11_token_uri)
                         printf(i == hr->pkcs11_token_uri ?
-                               "  Sec. Token: %s\n" :
+                               "PKCS11 Token: %s\n" :
                                "              %s\n", *i);
         }
 
+        if (hr->n_fido2_hmac_credential > 0)
+                printf(" FIDO2 Token: %zu\n", hr->n_fido2_hmac_credential);
```

Comment:
The contents is pretty opaque, it's usually a hash of some of the client data parameters we gave the token, and thus is highly app specific. It's not really usable for identification (since it isn't typically even stored on the FIDO devices, but re-generated by the device every time we talk to it, based on our parameters we pass in, and only after providing a PIN/requiring device touch). Hence, I see very little value in showing the thing, except for debugging. i.e. it's very different from a serial number of a pkcs#11 token, which is actually stored on the token and you can typically query before doing any operation or any authentication. it's also different from let's say a pgp key fingerprint, since that's something you want to pass around. 

Or in other words, it's more like a salt value for further calculations, and hence not too relevant to show. it would be a bit like showing the user the unix password hash as password, even though that's a pretty opaque thing too.